data/service/api/v1/summary.go

```diff
@@ -0,0 +1,113 @@
+package v1
+
+import (
+	"net/http"
+    "strconv"
+
+	//"github.com/tidepool-org/platform/data"
+	dataService "github.com/tidepool-org/platform/data/service"
+	//"github.com/tidepool-org/platform/page"
+	//"github.com/tidepool-org/platform/permission"
+	"github.com/tidepool-org/platform/request"
+)
+
+func SummaryRoutes() []dataService.Route {
+	return []dataService.Route{
+		dataService.MakeRoute("GET", "/v1/summary/:userId", Authenticate(GetSummary)),
+		dataService.MakeRoute("POST", "/v1/summary/:userId", Authenticate(UpdateSummary)),
+		dataService.MakeRoute("GET", "/v1/agedsummaries/:minutes", Authenticate(GetAgedSummaries)),
+	}
+}
+
+func GetSummary(dataServiceContext dataService.Context) {
+	res := dataServiceContext.Response()
+	req := dataServiceContext.Request()
+	dataClient := dataServiceContext.DataClient()
+
+	//details := request.DetailsFromContext(req.Context())
```

Comment:
Consider removing these commented out blocks for clarity, since the access permissions are checked by the router wrapping the function in `Authenticate`. Otherwise, a reader might wonder if we've explicitly disabled permission checks (which is not true).

Reply:
Actually, it looks like `Authenticate` only checks that some auth credential was provided (i.e. the commented out code). It does not appear to be check that the session token in the request is valid for this particular `userId`.

So unless I missed something, this needs more permission validation, similar to `./data/service/api/v1/users_datasets_get.go`.